pkg/upgrade/config.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	ControlPlane      = "control-plane"
```

Comment:
I'm not sure these technically need to live here vs `main.go`. It's probably a toss-up, although I doubt we'd move the `upgradeCluster` logic from `main.go` into the `pkg` code.

Wherever they live, I think it's nice to have a common word to group together related constants, so either ScopeControlPlane & ScopeMachineDeployment or ControlPlaceScope and MachineDeploymentScope. WDYT?

Reply:
I don't really car tbh. The other thing I've seen is to do like, `type Scope string` but I tend to find that more onerous than beneficial.

Will update to add scope to the suffix.